Makefile

```diff
@@ -48,7 +48,7 @@ generate-full-help-doc:
 	cargo run --bin doc-gen --features clap-markdown
 
 test: build-test
-	cargo test --workspace --exclude soroban-test
+	cargo test --workspace --exclude soroban-test --features additional-libs
```

Comment:
I wonder if we should include a test run without `additional-libs` too? I may punt on that for now, and consider how to do that when I implement the same for the ledger deps.

Reply:
Maybe we can have a separate job in the `rust.yml` workflow that had additional-libs enabled 🤔

Reply:
i think we should just update makefile to only run additional-libs tests

Reply:
> i think we should just update makefile to only run additional-libs tests

Does this change of adding `--features additional-libs` to the test command do that?

Reply:
I think so, yeah (but it needs to be a separate command)

Reply:
Are you thinking that there would be another command in the Makefile that just runs additional-libs tests?

Like, adding this:
```
additional-libs-test:
	cargo test --workspace --exclude soroban-test --features additional-libs
```
 And then would additional-lib-test be run as part of the main test command? or a separate workflow?

Reply:
I think just adding it to test command is fine

Reply:
just pushed up a change